monitor-translations/apache/create-urls-list.json

```
@@ -0,0 +1,11 @@
+{
+  "agentType": "TELEGRAF",
+  "monitorType": "http",
+  "name": "create-urls-list",
```

Since name needs to be unique across all these, how about
```suggestion
  "name": "apache-create-urls-list",
```